migration/process-migration/src/test/java/io/camunda/migration/process/it/AdapterTest.java

```diff
@@ -75,6 +76,9 @@ public abstract class AdapterTest {
   private static final OpensearchContainer<?> OS_CONTAINER =
       TestSearchContainers.createDefaultOpensearchContainer();
 
+  private static ObjectMapper esObjectMapper;
+  private static ObjectMapper osObjectMapper;
```

Comment:
❓ Do we need two different ones?

Reply:
Not great, but in these tests everything is "doubled" so I just kept two of them just in case, because they could have different configurations potentially.